LanguageFeatures/Inline-classes/static_analysis_inline_class_A09_t02.dart

```diff
@@ -22,10 +22,7 @@ extension Ex2 on int {
   String bar() => "Ex2.bar()";
 }
 
-inline class V {
-  final int id;
-  V(this.id);
-}
+inline class V(int id) {}
```

Comment:
Should this become `extension type` here and in the other tests below?

Reply:
Thank you for noticing! Fixed